paddle/fluid/framework/mixed_vector.h

```diff
@@ -386,13 +387,14 @@ template <typename T>
 class CPUVector : public std::vector<T, std::allocator<T>> {
  public:
   CPUVector() : std::vector<T>() {}
-  CPUVector(size_t count, const T &value = T())
+  explicit CPUVector(size_t count, const T &value = T())
```

Comment:
I'm wondering why we need a CPUVector? the mixed vector behaves same with std::vector.

Reply:
The mixed vector have a lock even in CPU single thread, that results in serious performance issue in Feed online service.

The `CPUVector` is a bugfix for that issue.

Reply:
Sorry I didn’t get the point, why it will have a lock？we didn’t use any mutex or sync primitives, is there any test result that verify the conclusion?